Closes i-RIC/prepost-gui#122

kskinoue0612 opened the pull request:
Function to export calculation result in *.tpo file format is implemented.

scharlton2 commented:
Hi Keisuke,

This branch didn't compile on my machine.  I think this:

`, const QVector2D& offset`

needs to be added to the end of

`PostZoneDataTpoExporter::exportToFile`

Thanks,
Scott

kskinoue0612 commented:
Hi, Scott

Thank you for testing, and I'm sorry for the issue...

I'm sure I've tested, so I don't know why. Anyway, I'll check it.

I have to participate a seminar today, and I think I can give you response about this on next Monday.

Thanks,
Keisuke

kskinoue0612 commented:
Hi Scott,

I've fixed the issue, and pushed a commit.

The reason that the pull request had an issue was that after merging the offset handling function to master, I've rebased the tpo_exporter branch on master, and did not test building. I'm sorry.

I think it's OK now. Please check this.

--
Keisuke
